README.md

```diff
@@ -98,7 +98,7 @@ spec:
   source:
     git:
       ref: "master"
-      uri: "https://github.com/openshift-psap/kvc-simple-kmod.git"
+      uri: "https://github.com/ebelarte/silly-kmod.git"
```

Comment:
1. This repo is private so I can't see its content :) 
2. @enriquebelarte What is the difference between "silly-kmod" and simple-kmod? can we keep using simple-kmod?
3. @bthurber If we keep "silly-kmod" we should probably move it into `openshift` org? WDYT?